.github/workflows/test.yml

```diff
@@ -49,8 +41,23 @@ jobs:
           CFLAGS: "-Werror -Wall -Wextra"
           MAXMINDDB_REQUIRE_EXTENSION: 1
 
-      - name: Test with tox
+      - name: Test with tox (vendored libmaxminddb)
+        run: tox
+        env:
+          MAXMINDDB_REQUIRE_EXTENSION: 1
+          MM_FORCE_EXT_TESTS: 1
+
+      - name: Install libmaxminddb
+        run: sudo apt install libmaxminddb-dev
+        if: matrix.platform == 'ubuntu-latest'
+
+      - name: Install libmaxminddb
+        run: brew install libmaxminddb
+        if: matrix.platform == 'macos-latest'
+
+      - name: Test with tox (system libmaxminddb)
```

Comment:
Hmm, wouldn't we have already built everything by this point though? That is, will we really use the system version? Probably I don't understand the process well enough.

Reply:
`tox` creates a fresh environment for each build it does.

Reply:
Ah ok. Do we even need the setup.py build step then? Maybe it's for -Werror in particular.

Reply:
See, e.g., [this build](https://github.com/maxmind/MaxMind-DB-Reader-python/actions/runs/6738912182/job/18319410926) that I just did.

Reply:
Yeah, I think that is there for that reason. I don't know that it is strictly necessary. Maybe it should just be a separate build or be added to static analyzer build.